transaction/core/src/amount/mod.rs

```diff
@@ -5,85 +5,67 @@
 //! Amounts are implemented as Pedersen commitments. The associated private keys
 //! are "masked" using a shared secret.
 
-#![cfg_attr(test, allow(clippy::unnecessary_operation))]
+// FIXME: Remove when prost-generated for `derive(Oneof)` has the necessary
+// doc comments: https://github.com/tokio-rs/prost/issues/237
+#![allow(missing_docs)]
```

Comment:
Prost 0.11 was released that I think addresses this.

Reply:
Cool